WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -265,3 +324,42 @@ fileprivate struct MediaSizingRow: ImmuTableRow {
         (cell.minValue, cell.maxValue) = MediaSettings().allowedImageSizeRange
     }
 }
+
+fileprivate class AppSettingsEditorFooterView: UITableViewHeaderFooterView {
+    static let height: CGFloat = 38.0
+    static let reuseIdentifier = "AppSettingsEditorFooterView"
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.axis = .horizontal
+
+        let label = UILabel()
+        label.font = UIFont.preferredFont(forTextStyle: .footnote)
+        label.text = NSLocalizedString("Editor beta release notes & bug reporting", comment: "Label for button linking to release notes and bug reporting help for the new beta Aztec editor")
+        label.textColor = WPStyleGuide.greyDarken10()
+        stackView.addArrangedSubview(label)
+
+        let button = UIButton(type: .custom)
```

Comment:
any reason for this to be a button? or can it be just an UIImageView?

At the moment you can tap it, but it doesn't do anything...